src/Aspire.Dashboard/Components/Dialogs/SettingsDialog.razor

```diff
@@ -15,7 +17,15 @@
             <FluentRadio Value="@ThemeManager.ThemeSettingDark">@Loc[nameof(Dialogs.SettingsDialogDarkTheme)]</FluentRadio>
         </FluentRadioGroup>
 
+        <FluentCombobox TOption="CultureInfo"
+                        Autocomplete="ComboboxAutocomplete.List"
+                        Immediate="true"
+                        Label="@Loc[nameof(Dialogs.SettingsDialogLanguage)]"
+                        Items="@_languageOptions"
+                        OptionText="@(c => c.NativeName.Humanize())"
```

Comment:
We could determine this based on the Accept-Language header or whether UI culture differs from culture, but `CultureInfo.CurrentUICulture` would just be set to english so that option would be selected. What do you think?